pkg/handler/crud.go

```diff
@@ -259,6 +279,74 @@ func (c *crud) DeleteFlag(params flag.DeleteFlagParams) middleware.Responder {
 	return flag.NewDeleteFlagOK()
 }
 
+func (c *crud) DeleteTag(params tag.DeleteTagParams) middleware.Responder {
+	t := &entity.Tag{}
+	t.ID = uint(params.TagID)
+
+	s := &entity.Flag{}
+	s.ID = uint(params.FlagID)
+
+	if err := getDB().Find(s).Association("Tags").Delete(t).Error; err != nil {
+		return tag.NewDeleteTagDefault(500).WithPayload(ErrorMessage("%s", err))
+	}
```

Comment:
You may also need `entity.SaveFlagSnapshot()` to capture tag deletion in flag history.